spec/activityDetection.spec.ts

```diff
@@ -0,0 +1,66 @@
+describe('activity detection', () => {
+
+    const originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+
+    beforeEach(() => {
+        jest.resetModules();
+        jest.useFakeTimers();
+    });
+
+    afterAll((done) => {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+        done();
+    });
+
+    it('should call `setWindowAndThreshold` correctly', () => {
+        const { activityDetection } = require('../src/app/activity-detection');
```

Comment:
any reason we are requiring here? Can't we use import at the beginning of the file?